src/binary.ts

```diff
@@ -16,6 +17,10 @@ export interface BinaryExtended {
   };
 }
 
+export interface UUIDExtended {
+  $uuid: string;
+}
```

Comment:
Might as well put this in the uuid.ts file, since we're encapsulating stuff in that module anyway. Super minor, up to you.